README.md

```diff
@@ -1,26 +1,26 @@
 # Firebase & AngularJS Starter Pack
 
-> [Firebase](https://www.firebase.com) is a scalable realtime backend that lets you build apps fast without managing servers.
+> [Firebase](https://www.firebase.com) is a powerful platform for your mobile and web applications that lets you build apps fast without managing servers.
 
 ## Introduction
-This starter pack is for anyone building a web app with Angular and Firebase. Firebase's realtime data store works well with [Angular](http://angularjs.org)'s two-way data binding and content rendering. This repository includes example applications that use some of the common patterns and best practices for integrating the two technologies.
+This starter pack is for anyone building a web app with Angular and Firebase. Firebase's realtime database works well with [Angular](http://angularjs.org)'s two-way data binding and content rendering. This repository includes example applications that use some of the common patterns and best practices for integrating the two technologies.
 
-We recommend taking the tutorial on both [Firebase](https://www.firebase.com/tutorial/) and [AngularJS](http://docs.angularjs.org/tutorial) before proceeding.
+We recommend taking the tutorial on both the [Firebase](https://www.firebase.com/tutorial/) database and [AngularJS](http://docs.angularjs.org/tutorial) before proceeding.
```

Comment:
suggestion: "the Firebase database" -> "the Firebase JavaScript client" I'd also make the whole text link to the tutorial